public/index.html

```diff
@@ -4,6 +4,7 @@
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-17134933-4"></script>
     <script src="https://kit.fontawesome.com/6be4547409.js" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/jquery-latest.min.js"></script>
```

Comment:
why do we need this? would it work if we add jQuery as a `package.json` dependency instead?

Reply:
@onursumer Adding jQuery to `package.json` is not working, I still got "$ is not defined" error

Reply:
Are you getting this error from `react-mutation-mapper`? Can be related to one of the latests updates inside the package.

Reply:
@leexgh I just did a quick debug, it is a problem with `react-mutation-mapper` package. Seems like the problem was introduced by this file https://github.com/cBioPortal/cbioportal-frontend/blob/3675a07a438ad31562b5945e22f71d1be8e7de55/packages/react-mutation-mapper/src/util/LabelUtils.tsx#L11. We should fix the issue there instead of adding a global import in genome-nexus.

Reply:
@leexgh I haven't tested it but this PR should fix the issue: cBioPortal/cbioportal-frontend#3985

Reply:
Tested locally, it seems working.